<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1321</width>
    <height>892</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Home</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(230, 230, 230);</string>
  </property>
  <widget class="QPushButton" name="Bt_Consultar_Aluno">
   <property name="geometry">
    <rect>
     <x>1140</x>
     <y>100</y>
     <width>161</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(180, 180, 180);</string>
   </property>
   <property name="text">
    <string>Consulta aluno</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Bt_Consultar_Assunto">
   <property name="geometry">
    <rect>
     <x>1140</x>
     <y>150</y>
     <width>161</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(180, 180, 180);</string>
   </property>
   <property name="text">
    <string>Consulta assunto</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Bt_Consultar_Atividade">
   <property name="geometry">
    <rect>
     <x>1140</x>
     <y>200</y>
     <width>160</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(180, 180, 180);</string>
   </property>
   <property name="text">
    <string>Consulta atividade</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tabela">
   <property name="geometry">
    <rect>
     <x>38</x>
     <y>96</y>
     <width>1091</width>
     <height>741</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string> Código</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string> Alunos </string>
    </property>
   </column>
   <column>
    <property name="text">
     <string> Atividade </string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Conclusão </string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Desempenho</string>
    </property>
   </column>
  </widget>
  <widget class="QPushButton" name="Bt_CadastroAssunto">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>40</y>
     <width>251</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(180, 180, 180);</string>
   </property>
   <property name="text">
    <string>Cadastro de Assunto</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Bt_CadastroAtividade">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>40</y>
     <width>251</width>
     <height>41</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>351</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(180, 180, 180);</string>
   </property>
   <property name="text">
    <string>Cadastro de Atividade</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Bt_CadastroDesempenho">
   <property name="geometry">
    <rect>
     <x>880</x>
     <y>40</y>
     <width>251</width>
     <height>41</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>351</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(180, 180, 180);</string>
   </property>
   <property name="text">
    <string>Cadastro de Desempenho</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Bt_CadastroAluno">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>40</y>
     <width>251</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(180, 180, 180);</string>
   </property>
   <property name="text">
    <string>Cadastro de Aluno</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
